### PR TITLE
Desktop-version-setup

### PR DESCRIPTION
**Pull request by sharon-odhiambo**
Add media queries for responsiveness
Check linter errors
Professional repo documentation

### CODE REVIEW

**Review by Ginohmk — CHANGES_REQUESTED** [review]
Hi @sharon-odhiambo @Hirwacardinalloic, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
you-are-near

## Highlights
- Linters passed :+1: 
- Good commit messages :100: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Code reviewer:
- If you find this feedback confusing in anyway, please reach out to me on slack `@kanu mike chibundu`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -1,413 +1,259 @@
...
-            <textarea
-              class="textarea"
-              id="msg"
-              name="user_message"
-              maxlength="500"
-              placeholder="Write your message here"
-              required
-            ></textarea>
+            <span><img src="./images/CSS.png" alt="css-icon">CSS</span></li>
+          </ul>
```
Comment (Ismailco) [inline index.html:215]: - [ ] As the previous reviewer mentioned kindly align this section properly so it looks similar to the Figma design.

|Your design|Figma design|
|-|-|
|Screenshot_20220610_181145|173090959-4726e422-35d7-4811-9861-9191a9879d3d|

File: index.html [hunk 2]
```diff
@@ -1,413 +1,259 @@
...
+        <li>
+          <label for="mail"></label>
+          <input type="email" id="mail"  placeholder="Enter Email" name="user_mail" required/>
+        </li>
+        <li>
+          <label for="msg"></label>
+          <textarea class="textarea" id="msg" name="user_message" placeholder="Write your message here" maxlength="500" required></textarea>
+        </li>
+        <li>
+        <button type="submit" class="but">Get In Touch</button>
```
Comment (Ismailco) [inline index.html:253]: - [ ] The buttons on your page buttons doesn't have any interaction, please make sure the colors are changed when the user hover over or clicks on the button

|Screenshot_20220610_181338|Screenshot_20220610_181349|
|-|-|
|Figma buttons design =>|173092219-080e5e0a-0542-4a16-be80-9db89c79fda1|

Comment (Ismailco) [inline index.html:253]: - [ ] [OPTIONAL] Please make sure the Submit button is centered on the form, this will make your design closer to the Figma design

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @sharon-odhiambo,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Sharon, 
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**